cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -214,15 +226,49 @@ install_cross_toolchain_pkg() {
   info "Configuring environment variables for cross-compilation"
   export PATH="/build/bin:${PATH}"
   export SYSROOT="/build/usr/x86_64-cros-linux-gnu"
-  export CC="x86_64-cros-linux-gnu-gcc"
+}
+
+set_compilation_env() {
```

Comment:
Can you please add an info() in the beginning of the function to show the container progress?